web/migrations/0057_forumtopic_github_issue_url_and_more.py

```
@@ -0,0 +1,23 @@
+    operations = [
+        migrations.AddField(
+            model_name="forumtopic",
+            name="github_issue_url",
+            field=models.URLField(blank=True, help_text="Link to related GitHub issue"),
+        ),
+        migrations.AddField(
+            model_name="forumtopic",
+            name="github_milestone_url",
+            field=models.URLField(blank=True, help_text="Link to related GitHub milestone"),
+        ),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)